src/HH4b/hh_vars.py

```diff
@@ -409,12 +410,13 @@
     "2023BPix": {"nom": 1.089, "down": 1.025, "up": 1.152},
 }
 jmsr_values["bbFatJetParTmassVis"]["JMS"] = {
-    "2022": {"nom": 1.009, "down": 1.005, "up": 1.012},
-    "2022EE": {"nom": 1.009, "down": 1.005, "up": 1.012},
-    "2023": {"nom": 0.928, "down": 0.915, "up": 0.942},
-    "2023BPix": {"nom": 0.928, "down": 0.915, "up": 0.942},
+    "2022": {"nom": 1.008, "down": 1.005, "up": 1.012},
```

Comment:
@cmantill so to confirm, you're using the other method in the skimmer, but you also updated the template-morphing numbers here?

Reply:
once we confirm the skimmer method works, i'd probably just stick to that method and remove this